definitions/infra-ibmmq_queue/definition.yml

```diff
@@ -19,6 +19,9 @@ synthesis:
       tags:
         clusterName:
 
+dashboardTemplates:
+  newRelic:
+    template: dashboard.json
 configuration:
-  entityExpirationTime: DAILY
+  entityExpirationTime: EIGHT_DAYS
```

Comment:
just double checking you want to increase the TTL of the entity.

Reply:
Changed to daily